starlette/staticfiles.py

```diff
@@ -157,11 +157,13 @@ def lookup_path(
         self, path: str
     ) -> typing.Tuple[str, typing.Optional[os.stat_result]]:
         for directory in self.all_directories:
-            full_path = os.path.realpath(os.path.join(directory, path))
+            original_path = os.path.join(directory, path)
```

Comment:
I haven't checked what the rest of the codebase uses, but I feel using the pathlib api is more modern and readable

Reply:
Yeah makes sense.